### PR TITLE
Admin - Resouces

### PR DESCRIPTION
**Pull request by strangiato**

### CODE REVIEW

**Review by rsriniva — APPROVED** [review]
LGTM for now. let's get this merged @jhurlocker and @strangiato

File: modules/chapter2/pages/resources.adoc [hunk 1]
```diff
@@ -1,12 +1,287 @@
...
+
+====
+
+== Removing an Unused Project
+
+Over time, projects may become abandoned as users leave your organization or move onto other initiatives. Old projects may need to be cleaned up and removed from the cluster in order to free up those resources.
+
+To completely remove a project from OpenShift, you must first delete all of the resources from that project before you will be able to delete the project.  
+
+The following `oc` commands can be used to help cleanup OpenShift AI related resources prior to deleting the project:
```
Comment (jramcast) [inline modules/chapter2/pages/resources.adoc:267]: Are the following commands equivalent to removing the project from the RHOAI dashboard?

**Review by jramcast — APPROVED** [review]
LGTM. I left a question regarding project removal.

Also, are we finally including a short GPU config lecture? I assume that will be done in a separate PR?

File: modules/chapter2/pages/resources.adoc (hunk 1 above)
Comment (strangiato) [reply modules/chapter2/pages/resources.adoc:267]: You can do most of this from the Dashboard.

I think adding something for how to do it from the UI is a good future enhancement.  I decided to leave it out just for the ability to get something done for this section before Monday.

Do we have a good place to add future this as a future todo?  I can add it to the issues on the repo here or would it make more sense to do something with it in Jira?

**Comment by strangiato** [comment]
> Also, are we finally including a short GPU config lecture? I assume that will be done in a separate PR?

I'm going to try and knock something out in the Installation Chapter.  I was a bit confused what the vision was for why GPU was included in this section, but if you have ideas please let me know (or lets throw it into some sort of backlog).

**Comment by jramcast** [comment]
> > Also, are we finally including a short GPU config lecture? I assume that will be done in a separate PR?
> 
> I'm going to try and knock something out in the Installation Chapter. I was a bit confused what the vision was for why GPU was included in this section, but if you have ideas please let me know (or lets throw it into some sort of backlog).

I assume we could just give them a short high-level view of the general steps that an admin should follow to enable GPUs. Only narrative, we are not doing hands-on exercise for this part in the workshop.

For example, a couple of introductory paragraphs + an ordered list of steps might be enough for the workshop? Something like the following draft maybe (feel free to adjust my content, I have almost no idea of this topic :rofl: )?

```
One of the key features of RHOAI is the ability to provide data scientists with access to ready-to-use GPU resources.
This capability relieves data scientists from the hassle of configuring GPU drivers and library versions in their own environments.

As an administrator, you must configure GPU resources to make them available for RHODS regular users.
To configure GPUs, follow these steps:

1. If working on bare metal, on-premise, make sure that you install drivers on your compute nodes, blah blah...
2. If working on public clouds...
3. Install the Node Feature Discovery Operator
4. Install the GPU Operator
5. ....
```

In any case, this PR can be merged as is, if you have not started adding GPU content to this PR, I suggest we merge this one and use a different PR for the GPU content.

**Pull request merged**